packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -27,35 +36,40 @@ const defaultValue = 'This is a caption';
 const groupIdentifier = 'CAPTION VARIANTS';
 
 <Caption>
-  {select(label, LANGUAGE_VARIANTS, defaultValue, groupIdentifier)}
+  {select(label, LANGUAGE_VARIANTS, LANGUAGE_VARIANTS.English, groupIdentifier).text}
 </Caption>
 ```
 
 ### inputProvider
 
-N.B. the number of elements in the array should equal the number of arguments taken
-by the render function.
-
 ```jsx
+import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import Caption from '@bbc/psammead-caption';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 
 storiesOf('Caption', module)
   .addDecorator(withKnobs)
   .add(
     'default',
     inputProvider(
-      ['caption', 'visually hidden text'],
-      (captionText, vhText) => (
-        <Caption>
-          <VisuallyHiddenText>{vhText}</VisuallyHiddenText>
+      [
+        { name: 'caption', defaultText: 'Students sitting an examination' },
+        { name: 'offscreen text', defaultText: 'Image Caption, ' },
```

Comment:
Where does the `name` (EG: "offscreen text") get displayed?

Reply:
In the list of knobs; there will be a 'Select Language' knob, a 'Content for caption' knob and a 'Content for offscreen text' knob, and these names must (ie. required by storybook) be unique. Also see https://github.com/bbc/psammead/pull/468/files/b550e3580bd963c1919b8d7e297329caec61944d#diff-c2e7d4722b8df6ba5cc5dfab8c49be40R20